File: Project/C++/Project1/Notebook.cpp
#include "Notebook.h"
int Notebook::ammount = 0;
Notebook::Notebook(std::string name, int price, std::string description, int stock, int id, int sections)
{
	setName(name);
	setPrice(price);
	setDescription(description);
	setStock(stock);
	setId(id);
	setObjType("notebook");
	this->sections = sections;
	this->ammount++;
}

std::string Notebook::getDetails()
{
	return "Sections: " + std::to_string(sections) + ".";
}

File: Project/Java/src/eshop/Notebook.java
package eshop;

public class Notebook extends Item{
    private int sections;
    private static int count = 0;

    Notebook(String name, double price, String description, int stock, int id, int sections){
        setName(name);
        setPrice(price);
        setDescription(description);
        setStock(stock);
        this.sections = sections;
        count++;
    }
    public String getDetails(){
        return "Sections: " + sections;
    }
    public static int getCount() {
        return count;
    }

}
